src/api/flow.ts

```diff
@@ -210,10 +154,22 @@ export function createFlowGenerator(name: string, generator: Function) {
                 if (ret.done) return resolve(ret.value)
                 // TODO: support more type of values? See https://github.com/tj/co/blob/249bbdc72da24ae44076afd716349d2089b31c4c/index.js#L100
                 if (!ret.value || typeof ret.value.then !== "function")
-                    fail("Only promises can be yielded to asyncAction, got: " + ret)
+                    return fail("Only promises can be yielded to asyncAction, got: " + ret)
```

Comment:
This is different from the default async/await behavior

Reply:
Fixed, using `Promise.resolve(ret.value)` now

Reply:
That sounds good, though it's a matter of choice mostly